pom.xml

```diff
@@ -98,8 +98,9 @@
         <elasticsearch.version>5.0.2</elasticsearch.version>
         <delight-nashorn-sandbox.version>0.2.1</delight-nashorn-sandbox.version>
         <!-- IMPORTANT: If you change the version of the kafka client, make sure to synchronize our overwritten implementation of the
-        org.apache.kafka.common.network.NetworkReceive class in the application module. It addresses the issue https://issues.apache.org/jira/browse/KAFKA-4090. -->
-        <kafka.version>2.8.0</kafka.version>
+        org.apache.kafka.common.network.NetworkReceive class in the application module. It addresses the issue https://issues.apache.org/jira/browse/KAFKA-4090.
```

Comment:
Are you sure the NetworkReceive class was not changed?

Reply:
absolutely sure. It has not been modified for 3 years.
![image](https://user-images.githubusercontent.com/79898499/174847858-99b184e1-42f6-4133-a462-abbcb1ba817e.png)